src/Cogs/ump.py

```diff
@@ -643,6 +644,40 @@ async def set_sheet_id(self, ctx, sheet_id):
                                        'https://docs.google.com/spreadsheets/d/%s' % sheet_id)
                     break
 
+    @commands.command(brief='Game setup',
+                      description='')
+    @commands.has_role(ump_role)
+    async def setup(self, ctx, league, ump2: discord.Member = None, ump3: discord.Member = None, ump4: discord.Member = None,
+                    ump5: discord.Member = None, ump6: discord.Member = None):
+        await ctx.message.add_reaction('<a:baseball:872894282032365618>')
```

Comment:
<a:baseball:872894282032365618> looks a static variable. Any change this could be moved to a variables file or something, in case this reaction is ever changed/deleted?

Reply:
it's a custom emoji from my test server, which you can use because bots apparently just include nitro and can use custom emojis everywhere. If i ever remove the emoji or the bot from my server it would probably break though so... maybe not a bad idea...

Reply:
Isn't there a unicode baseball emoji at this point?